### ISSUE TITLE
HOA Abandonment?

### ISSUE THREAD

**Issue by abcarroll** [K-Phoen/rulerz#116]
Hi there,

So from a discussion that began regarding hoa/console and it's abandonment, I was directed towards [a tweet by Ivan E](https://twitter.com/mnt_io/status/1207979302769348610).  

I don't use hoa/console currently, but I do use your rulerz project.... I also noticed you are, on github, a part of the HOA project.  I was just wondering if you are aware of any future support or non-support for the hoa/ruler library that your rulerz project depends on?

Thanks!

**Comment by K-Phoen** [comment]
Hi,

I was kind of involved at some point in the past with Hoa but not really anymore. As such, I don't really know if they plan on supporting their ruler or not.
When it comes to the impact on RulerZ itself, I have no plan right now to migrate to another library for two main reasons:
* hoa/ruler does the job
* I don't have the time to actively maintain RulerZ (and I don't even work with PHP anymore :D), which is why RulerZ hasn't had a stable release yet.

TL;DR: my plan is "no-op".

**Comment by SerjoA** [comment]
thank you for the library btw, its awesome and well built and documented
